lucida/commandcenter/app.py

```diff
@@ -43,7 +42,7 @@ def flask_listener():
                 threaded=True)
 
 def web_socket_listener():
-    print 'Start web socket at ' + str(cmd_port)
+    print 'Start web socket at 8081' 
```

Comment:
Yeah, or we could start the Lucida core from the GUI. Either way, I think you are correct in that for now it isn't too significant.